backend/dissemination/management/commands/materialized_views.py

```diff
@@ -0,0 +1,26 @@
+from django.core.management.base import BaseCommand
+from dissemination import api_versions
+
+refresh_views = """
+REFRESH MATERIALIZED VIEW dissemination_combined;
```

Comment:
I think we need to use `CONCURRENTLY` here or the refresh will rebuild the _entire_ MV, which blocks all interactions until it's done. https://www.postgresql.org/docs/15/sql-refreshmaterializedview.html